.eslintrc.yml

```diff
@@ -0,0 +1,60 @@
+extends:
+- airbnb-base
+- next
+
+env:
+  browser: true
+  es2021: true
+  es6: true
+  jest: true
+
+globals:
+  google: readonly
+  module: readonly
+  process: readonly
+
+parserOptions:
+  ecmaVersion: 8
+  sourceType: module
+  ecmaFeatures:
+    jsx: true
+
+rules:
+  arrow-body-style: off
+  arrow-parens: off
+  func-names: warn
+  func-style: off
+  max-len: warn
```

Comment:
[This rule](https://eslint.org/docs/rules/max-len) is another one I don't have a preference for, in terms of leaving it as an error or turning it into a warning. My concern is that there might be a really long link or block of text or something that unavoidably exceeds the max length.

Reply:
Let's do enforced 100 char line length